### PR TITLE
Add ability to change the active team in the save file

### PR DESCRIPTION
**Pull request by martinbutt**
Adds a new `-t` flag that allows you to change your team in the save file. This is particularly useful for starting the game with a higher league team.

Instead of trying to work out how to copy the correct stadium for the team, I opted to generate a stadium depending on the team's starting league.

### CODE REVIEW

File: main.cc [hunk 1]
```diff
@@ -2503,4 +2534,160 @@ void soup_up(int player) {
 		player.morl = 8;
 	}
 }
+
+void change_club(int new_club_idx, int new_player) {
+	struct gamea::manager &manager = gamea.manager[new_player];
```
Comment (eb4x) [inline main.cc]: Just `player`. (It's at most a two-player hot-seat game, and you're either player0 or player1.)

File: main.cc [hunk 2]
```diff
@@ -2503,4 +2534,160 @@ void soup_up(int player) {
...
+			}
+
+    		for (int i = 0; i < 4; ++i) {
+    			manager.stadium.capacity[i].seating = 10000;
+    			manager.stadium.capacity[i].terraces = 0;
+    			manager.stadium.conversion[i].level= 2;
+    			manager.stadium.area_covering[i].level= 3;
+    		}
+
+		    manager.price.league_match_seating = 15;
```
Comment (eb4x) [inline main.cc]: The indentation seems off.

File: main.cc [hunk 3]
```diff
@@ -578,19 +579,29 @@ void dump_player(struct gamec::player &player);
 void print_player_name(int16_t idx, bool newline = true);
 
 void print_help(char *command) {
-	fprintf(stderr, "Usage: %s -g 1-8 [ /path/to/saves/ ]\n", command);
+	fprintf(stderr, "Usage: %s -g 1-8 [-t 0-75] [-s] [-h] [ /path/to/saves/ ]\n", command);
```
Comment (eb4x) [inline main.cc]: Shouldn't it be `0-113`?

**Review by eb4x — CHANGES_REQUESTED** [review]
Wow, I'm flabbergasted someone else actually ended up working with this crusty old code :sweat_smile:

Generating the stadium based on league seems like a fair enough approach to me. The code-indentation seems a bit off, but if it gets sorted, I'd happily merge this :+1:

(And in any case, thanks for letting me know about this project seeing some use.)

**Comment by eb4x** [comment]
Looks good to me, thanks!

**Review by eb4x — COMMENTED** [review]
LGTM.

**Comment by martinbutt** [comment]
Thank you for your effort on this. It's been a lot of fun to play with.

**Pull request merged**